README.md

```diff
@@ -1225,14 +1227,20 @@ An Azul build on Gitlab runs the `test`, `terraform`, `deploy`, `subscribe` and
 feature branches is `sandbox`, the protected branches use their respective
 deployments.
 
-## 9.1 The Sandbox Deployment
+## 9.1 CI/CD System Links
+
+- [SingleCell Dev CI/CD Instance](https://gitlab.dev.singlecell.gi.ucsc.edu/)
+- [Allspark Dev CI/CD Instance](https://allspark.dev.data.humancellatlas.org/)
```

Comment:
We don't use Allspark. Azul has a separate Gitlab deployment for HCA: https://gitlab.dev.explore.data.humancellatlas.org/azul/azul

and likewise prod HCA:
https://gitlab.explore.data.humancellatlas.org/azul/azul/

Reply:
oh interesting, IIRC there was a copy of Azul within allspark, maybe its not being used, ill omit it from thee list.